### DISCUSSION TITLE
Missing JavaScripts, CSS and Fonts

### DISCUSSION THREAD

**Discussion by rlee2015** [Dotnet-Boxed/Templates#62] [Q&A]
Hi Rehan,

I can run the project in ASP.NET 5 MVC 6 (RC1 Update 1, version 2.9.4) without any problems however I found pages are not rendering properly due to the following missing files. 

/wwwroot/js/
- bootstrap.js
- jquery.js
- jquery-validate.js
- jquery-validate-unobtrusive.js
- modernizr.js
- site.js

/wwwroot/css/
- font-awesome.css
- site.css

Also, glyphicons-halflings-regular.\* are missing as well under /wwwroot/Content/ folder & sub-folders.

There is a warning (CS1998) in line 191 in FeedService.cs. Here is the description in VS2015.
"This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread."

Best Regards,

Richard

**Comment by RehanSaeed** [comment]
You need to run the gulfile.js pipeline to get the CSS, JS and fonts to show up. Use the task runner window to do this.

That warning is intentional, it's for you to fill in that method so that the warning goes away.
